Clarify and add details to documents

River-b opened the pull request:
Add details to several documents and clarify other sections.
Added release information to the History document.

discip commented on Documentation/Menu.md:
```diff
@@ -42,7 +42,7 @@ Some tips will have an offset on their readings, to calibrate this out, perform
 1. Connect power to your iron
 2. **Critical: make sure the tip & handle is at room temperature** (wait a good while after using the iron before calibration)
-3. Using (`-/B`) Scroll down to Advanced menu > press (`+/A`) > Calibrate temperature? > press (`+/A`)
+3. Using (`-/B`) scroll down to Advanced settings > press (`+/A`) to select > then (`-/B`) to scroll to Calibrate temperature? > press (`+/A`) until you see ...
```
```suggestion
3. Using (`-/B`) scroll down to Advanced settings > press (`+/A`) to select > then (`-/B`) to scroll to Calibrate temperature? > press (`+/A`) until you see **`...`**
```

River-b replied:
will change thanks this makes that much more visable.

River-b commented:
Could I get Clarification on  Temperature.md

Is this a Typo...  what does outside of this **bad**  refer to?

"For example, on a digital Weller unit that Ralim has, if set to 350°C, it will regulate to within around +/- 3°C but not indicate you are **outside of this bad** until you exceed +/- 5°C. "

River-b reviewed (commented):
My only question is for the **Temperature.md** file. 
I'm not sure what this sentence means or if there is simply a Typo of the word "**bad**"

" it will regulate to within around +-3 °C but not indicate you are outside **of this bad** until you exceed +-5 °C"

is this meant to say something like...

"it will regulate to within around +/- 3 °C but not indicated you are **outside of the margin of error** until you exceed +/-5 °C. This give the illusion that it's holding the temperature perfectly when in actuality it's moving around as well."

River-b commented:
-  I changed Temperature.md to "out of margin of error"

- changed Flashing page to show **V2  has no update** right now because we are getting at least 2-3 people daily  in  Chat  saying they can't seem to update it and think there is a problem with their Iron.  I think it would help to get that approved to display soon since chat represents only a small % of total new users, who may come to Github, but not even be members of chat. thanks

discip commented on README.md:
```diff
@@ -20,16 +20,17 @@ This project is considered feature complete for use as a soldering iron, _so ple
-|     TS80P | ❌  | ✔️  | ✔️  | ❌  | ❌  |   ✔️    |     ✔️      |
 
-Please note that Miniware started shipping TS100's using cloned STM32 Chips. While these do work with IronOS, their DFU bootloader works terribly, and it is hard to get it to successfully flash larger firmware images like IronOS without timing out. This is the main reason why the TS100 is _no longer recommended_.
+|   Device  | DC | QC | PD | EPR | BLE | Battery | Recommended |
+|----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
+|     MHP30 |❌ | ❌ | ✔️ | ❌ | ❌ |   ❌    |     ✔️     |
+| Pinecil V1 |✔️ | ✔️ | ✔️ | ❌ | ❌ |   ✔️   |     ✔️     |
+| Pinecil V2 |✔️ | ✔️ | ✔️ | ✔️ | ✔️ |   ✔️   |     ✔️     |
+|     TS100 |✔️ | ❌ | ❌ | ❌ | ❌ |   ✔️    |     ❌     |
+|      TS80 |❌ | ✔️ | ❌ | ❌ | ❌ |   ✔️    |     ❌     |
+|     TS80P |❌ | ✔️ | ✔️ | ❌ | ❌ |   ✔️    |     ✔️     |
```
```suggestion
|   Device   | DC | QC | PD | EPR | BLE | Battery | Recommended |
|-----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
|      MHP30 |❌ | ❌ | ✔️ | ❌ | ❌ |   ❌    |     ✔️     |
| Pinecil V1 |✔️ | ✔️ | ✔️ | ❌ | ❌ |   ✔️    |     ✔️     |
| Pinecil V2 |✔️ | ✔️ | ✔️ | ✔️ | ✔️ |   ✔️    |     ✔️     |
|      TS100 |✔️ | ❌ | ❌ | ❌ | ❌ |   ✔️    |     ❌     |
|       TS80 |❌ | ✔️ | ❌ | ❌ | ❌ |   ✔️    |     ❌     |
|      TS80P |❌ | ✔️ | ✔️ | ❌ | ❌ |   ✔️    |     ✔️     |
```
